src/features/ShowMovie/ui/DatesNav.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, useState } from 'react';
+
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import clsx from 'clsx';
+
+import { mapDayToLetter, mapMonthToLetter } from 'shared/utils/dateUtils';
+
+import { Date } from './Date';
+
+interface DatesNavProps {
+    dates: Date[];
+}
+const visibleItemsCount = 7;
+
+export const DatesNav: FC<DatesNavProps> = ({ dates }) => {
+    const [startIndex, setStartIndex] = useState(0);
+
+    const shiftLeft = () => {
+        if (startIndex > 0) {
+            setStartIndex(startIndex - 1);
+        }
+    };
+
+    const shiftRight = () => {
+        if (startIndex + visibleItemsCount < dates.length) {
+            setStartIndex(startIndex + 1);
+        }
+    };
+
+    return (
+        <div className="h-[150px] font-effra bg-gray-200 flex gap-16 items-center justify-center max-w-full">
+            <ArrowBackIosIcon
+                className={clsx(
+                    'cursor-pointer justify-self-start',
+                    dates.length > 7 ? '!block' : '!hidden',
```

Comment:
```suggestion
                    dates.length > visibleItemsCount ? '!block' : '!hidden',
```
try to avoid such magic numbers, use constants instead